index.js

```diff
@@ -0,0 +1,2 @@
+require('angular-xml.js');
+module.exports = 'xml';
```

Comment:
This seems odd to me, I'm no browserify expert, but in the common.js world, requiring this module would just return a string.